### PR TITLE
Update initializers.py

### PR DESCRIPTION
**Pull request by zhongzyd**
According to [Understanding the difficulty of training deep feedforward neural networks](http://jmlr.org/proceedings/papers/v9/glorot10a/glorot10a.pdf) and the codes, it is sqrt(2. / (in + out))

### CODE REVIEW

**Comment by drpngx** [comment]
@sguada what do you think?

**Comment by sguada** [comment]
Not clear to me the paper mentioned that.

**Comment by zhongzyd** [comment]
@sguada The paper mentioned that in (eq.12). 
The variance of the weights is  2. / (in + out) , so the standard deviation should be sqrt(2. / (in + out)).

**Comment by frankchn** [comment]
ping @sguada

**Comment by sguada** [comment]
The normalized initialization is defined in eq. (16) 
`sqrt(6) / sqrt(n_in + n_out)`
which equivalent to 
`sqrt(3 / ((n_in + n_out)/2))`

**Comment by zhongzyd** [comment]
@sguada 
Eq. (16) uses uniform distribution.  
If **W** is sampled from a uniform distribution in the range **[-sqrt(6) / sqrt(n_in + n_out), sqrt(6) / sqrt(n_in + n_out)]**, then it has a standard deviation of **sqrt(2) / sqrt(n_in + n_out)**.
So when **W** is sampled from a **normal distribution**, the standard deviation should be **sqrt(2) / sqrt(n_in + n_out)**. 

You could check the code in **https://github.com/tensorflow/tensorflow/blob/r1.2/tensorflow/contrib/layers/python/layers/initializers.py** .
If **mode == 'FAN_AVG'**, it uses a standard deviation of **sqrt(2) / sqrt(n_in + n_out)** for normal distribution. Because it is sampled from a **truncated normal distribution**, so it multiply the factor with 1.3 to keep the standard deviation at **sqrt(2) / sqrt(n_in + n_out)**.

**Comment by sguada** [comment]
@zhongzyd Thanks for the clarification, I miss understood your previous comment.

**Review by sguada — COMMENTED** [review]
Looks good

**Review by sguada — APPROVED** [review]

**Comment by zhongzyd** [comment]
@sguada Thanks.

**Comment by frankchn** [comment]
Jenkins, test this please.
